src/program/media/item.py

```diff
@@ -17,6 +18,22 @@
 from ..db.db_functions import blacklist_stream, reset_streams
 from .stream import Stream
 
+class ShowMediaType(Enum):
+    """Show media types"""
+    Show = "show"
+    Season = "season"
+    Episode = "episode"
+
+class MovieMediaType(Enum):
+    """Media types"""
+    Movie = "movie"
+
+class MediaType(Enum):
+    """Combined media types"""
+    Show = ShowMediaType.Show.value
+    Season = ShowMediaType.Season.value
+    Episode = ShowMediaType.Episode.value
+    Movie = MovieMediaType.Movie.value
```

Comment:
we dont need these classes lol, no need to make it more complicated than it has to be! ;)